lite/valor_lite/detection/annotation.py

```diff
@@ -15,6 +15,10 @@ class BoundingBox:
     scores: list[float] = field(default_factory=list)
 
     def __post_init__(self):
+        if len(self.scores) == 0 and len(self.labels) != 1:
+            raise ValueError(
+                "If no scores are defined, then this is a ground truth and a single label requirement is enforced."
```

Comment:
can you change this error message to be more explicit? something like "Ground truths must be defined with no scores and a single label. If you meant to define a prediction, then please include one score for every label provided."